django/santropolFeast/member/templates/client/base.html

```diff
@@ -38,7 +38,7 @@
                                 minDate: new Date(
                                     today.getFullYear(),
                                     today.getMonth(),
-                                    today.getDate() + 1),
```

Comment:
I wanted to make sure the change was processed, by preventing the user to select the current day.
But you address this in the views, by processing directly the changes of the current day, so that should do the trick.